helm/linkerd2-multicluster-app/Chart.yaml

```diff
@@ -1,23 +1,23 @@
-apiVersion: "v1"
-appVersion: stable-2.11.4
+apiVersion: v2
+appVersion: stable-2.12.2
+annotations:
+  application.giantswarm.io/team: team-cabbage
 description: |
   The Linkerd-Multicluster extension contains resources to support multicluster
   linking to remote clusters
-home: https://linkerd.io
+home: https://github.com/giantswarm/linkerd2-multicluster-app
+icon: https://linkerd.io/images/logo-only-200h.png
+dependencies:
+  - name: partials
+    version: 0.1.0
 keywords:
   - service-mesh
 kubeVersion: ">=1.17.0-0"
 name: "linkerd2-multicluster-app"
 sources:
+  - https://github.com/giantswarm/linkerd2-multicluster-app/
```

Comment:
I think sources can stay as is since we're trying to align the app to upstream and most of our other apps also just mention upstream URLs here.